_pages/programs.md

```diff
@@ -41,46 +41,53 @@ I moved pyOpenSci to a fiscal sponsor. I am now Executive Director of the projec
 and working to build a diverse community around Python open source software in 
 support of Open Science. 
 
-<a href="https://www.pyopensci.org/blog/new-executive-director-leah-wasser">Learn more <i class="fas fa-arrow-circle-right" aria-hidden="true"></i></a>
+<a href="https://www.pyopensci.org/blog/new-executive-director-leah-wasser">pyOpenSci Website <i class="fas fa-arrow-circle-right" aria-hidden="true"></i></a>
 
 <!-- ADD BLOG -->
-
 </div>
 
-## Previous Programs
 
-<div class="feature__wrapper">
-    <h3 class="archive__item-subsection">Building Earth & Environmental Data Science Capacity at Tribal & Hispanic Serving Colleges</h3>
-            <p class="page__meta">
-            <span class="page__meta-readtime">
-            <i class="fas fa-rocket" aria-hidden="true"></i>
-            NSF Funded: 2019-2022
-            </span>
-        </p>
-    <div class="feature__item--left">
-        <div class="archive__item">
-            <div class="archive__item-teaser">
-                <img src="/images/women-coding-unsplash.jpg" alt="customizable">
-            </div>
-            <div class="archive__item-body">
-                <div class="archive__item-excerpt">
-                    <p>I designed a program
-                    that 1) Provided training and career development in earth and environmental
-                    data science to undergraduate students at several institutions 2)
-                    built capacity for faculty to teach these skills at their institutions.
-                    The program was supported by a 1.3 million dollar NSF award in 2019. 
-                    Program Design Collaborator: <i>Jenny Palomino</i>
-                    </p>
-                </div>
-                <p><a href="" class="btn ">RELATED: Lessons Learned From The EDSC</a></p>
-            </div>
-        </div>
-    </div>
+
+<div class="notice-white">
```

Comment:
this all should really be data and parsed via a template. Once i land on the final template...